internal/api/deployments.go

```diff
@@ -74,7 +74,7 @@ type DeploymentUpdate struct {
 	EnforceParameterSchema bool                   `json:"enforce_parameter_schema,omitempty"`
 	Entrypoint             string                 `json:"entrypoint,omitempty"`
 	JobVariables           map[string]interface{} `json:"job_variables,omitempty"`
-	ManifestPath           string                 `json:"manifest_path"`
+	ManifestPath           string                 `json:"manifest_path,omitempty"`
```

Comment:
This is the fix for https://github.com/PrefectHQ/terraform-provider-prefect/issues/356 specifically.